src/AbstractState.cpp

```diff
@@ -127,14 +127,17 @@ AbstractState * AbstractState::factory(const std::string &backend, const std::ve
 
     std::map<backend_families,shared_ptr<AbstractStateGenerator> >::const_iterator gen, end;
     get_backend_library().get_generator_iterators(f1, gen, end);
+    std::cout << "after get_generator_iterators" << std::endl; // !!! remove
```

Comment:
Can you please remove these lines?

Reply:
I think I removed them all. At least I went through and tried to delete them all before submitting the pull request. I didn't see this line in the file anymore. Are you still seeing them?

Reply:
Yeah, I was looking at intermediate commit.